### PR TITLE
Cleanup sessions via SQL query instead of SQL function

### PR DESCRIPTION
**Pull request by ricardoasmarques**
## What does this PR change?

With this PR, sessions cleanup will be done by an SQL query instead of an SQL function, which results in less code, easier to maintain, and no more limitations on the number of sessions removed per execution.

The following items were removed in this PR:
- SQL function `pxt_session_cleanup`
- Configurations `java.session_delete_batch_size` and `java.session_delete_commit_interval`

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Fixes: https://github.com/SUSE/spacewalk/issues/13721

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by rjmateus — APPROVED** [review]
LTGM
Just one nitpik

**Comment by rjmateus** [comment]
@ricardoasmarques I think this should be backported to manager 4.1 and 4.0

File: schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+drop function if exists pxt_session_cleanup
+   (bound_in in numeric, commit_interval in numeric,
+    batch_size in numeric, sessions_deleted in numeric);
```
[the remark this answers is not in the document]
Comment (ricardoasmarques) [reply schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql]: done

File: schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+drop function if exists pxt_session_cleanup
```
Comment (juliogonzalez) [inline schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql:1]: Please move this file to 4.2.7-to-4.2.8, as I had to tag for submitting Beta1.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql (hunk 2 above)
Comment (juliogonzalez) [reply schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql:1]: And fix the conflicts for the spacewalk-java changelog :-)

Comment (ricardoasmarques) [reply schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql]: @juliogonzalez I've now moved it to `4.2.7-to-4.2.8`

**Review by juliogonzalez — APPROVED** [review]

File: schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql (hunk 2 above)
Comment (juliogonzalez) [reply schema/spacewalk/upgrade/susemanager-schema-4.2.6-to-susemanager-schema-4.2.7/802-drop-pxt_session_cleanup.sql]: Thanks. It's merged now :-)

**Pull request merged**